### PR TITLE
Add support for feature backup cancel

### PR DESCRIPTION
**Pull request by svt-vverma**
### Description
Add support for feature backup cancel

### Issues Resolved
None

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: simplivity/resources/backups.py [hunk 1]
```diff
@@ -213,5 +213,17 @@ def rename(self, new_name, timeout=-1):
...
+        """Cancels the specified running backup
+        Args:
+          timeout: Time out for the request in seconds.
 
+        Returns:
+          object: Backup object.
+        """
+        resource_uri = "{}/{}/cancel".format(URL, self.data["id"])
+        self._client.do_post(resource_uri, None, timeout)
+        self.__refresh()
```
Comment (sijeesh) [inline simplivity/resources/backups.py:264]: Will refresh work if the cancel operation is successful?

Comment (svt-vverma) [reply simplivity/resources/backups.py:264]: Yes refresh will work, if the cancel operation on backup whose creation is in-progress.
Object is created in the backend and the status for this object is "cancelled". 
If the cancel operation is not successful the POST command will throw the exception.

**Comment by sijeesh** [comment]
@svt-vverma, resolve conflicts

File: endpoints-support.md [hunk 2]
```diff
@@ -14,6 +14,14 @@ Refer SimpliVity REST API doc for the resource endpoints documentation [HPE Simp
 |<sub>/backups/{bkpId}/lock</sub>                                  |POST    |
 |<sub>/backups/{bkpId}/rename</sub>                                |POST    |
 |<sub>/backups/{bkpId}/restore</sub>                               |POST    |
+|<sub>/backups	</sub>                                                                    |GET       |
```
Comment (sijeesh) [inline endpoints-support.md]: remove duplicate entries

**Review by sijeesh — CHANGES_REQUESTED** [review]

File: endpoints-support.md (hunk 2 above)
Comment (svt-vverma) [reply endpoints-support.md]: Removed the duplicate entries and corrected the sequence.

**Review by sijeesh — APPROVED** [review]
LGTM 👍

**Pull request merged**